app/assets/stylesheets/smart_answers.scss

```diff
@@ -41,6 +41,14 @@ $is-ie: false !default;
     min-height: 0px;
   }
 
+  .descriptive__text a:focus {
```

Comment:
Could be worth using `@include govuk-focused-text;` here

Reply:
Ah, of course. Good shout, thanks. Are you happy to approve?